ecgtools/parsers/cmip.py

```diff
@@ -74,6 +74,7 @@ def parse_cmip6(file):
             attributes['init_year'] = init_year
             attributes['start_time'] = start_time
             attributes['end_time'] = end_time
+            attributes['time_range'] = f'{start_time}-{end_time}'
```

Comment:
Some variables in CMIP6 are time invariant and are made available under the `fixed` frequency. So, we may want to set `time_range` to `None` when `start_time` and `end_time` are None.